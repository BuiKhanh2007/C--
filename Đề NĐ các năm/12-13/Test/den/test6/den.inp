89 120
63 157
49 54
180 374
6 170
55 303
77 115
166 323
