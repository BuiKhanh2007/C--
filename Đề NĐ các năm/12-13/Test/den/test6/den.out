353
1042
57
2833
1717
3190
433
2237
